package.json

```diff
@@ -56,7 +55,6 @@
     "eslint-plugin-markdown": "^1.0.0-beta.3",
     "eslint-plugin-prettier": "^3.0.1",
     "eslint-plugin-react": "7.13.0",
-    "flow-bin": "^0.98.1",
```

Comment:
I don't think we can remove flow check here. We can move flow check to CI though it should be done in another PR.